### PR TITLE
Sort exported data

### LINKED ISSUE #490: bar export data not sorted by time
Line export data is sorted by meter an time. Bar is sorted by meter but not time. While the user could easily do this (if they know how in a spreadsheet), we should do it to be nice and to be consistent. The sorting should be by start time.

### PR DESCRIPTION
**Pull request by truongdd03**
# Description

Sorts the data before exporting to csv.

Fixes #490

## Type of change

- [ ] Note merging this changes the node modules
- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request

## Limitations

There is no other issue.

### CODE REVIEW

File: containers/web/Dockerfile
Comment (huss) [inline containers/web/Dockerfile:8]: I don't think we should include this change in the PR. It is something that needs more testing and will be in a separate PR. This also appears to be in PR #668.

File: src/client/app/containers/ExportContainer.ts
Comment (huss) [inline src/client/app/containers/ExportContainer.ts:18]: All these changes seem to be from PR #668. They either should be removed or we need to wait until that clears before we merge this.

File: src/client/app/types/readings.ts
Comment (huss) [inline src/client/app/types/readings.ts:39]: From PR #668 so same as others.

File: src/client/app/utils/exportData.ts
Comment (huss) [inline src/client/app/utils/exportData.ts:18]: From PR #668 so like others.

File: src/server/test/web/obviusTest.js
Comment (huss) [inline src/server/test/web/obviusTest.js:41]: From PR #668 so like others.

**Review by huss — COMMENTED** [review]
The PR works as expected. The check is failing put same issue as PR #668 so should be okay once that is fixed (but will need to be merged so it fixes it here too). The only issue is inclusion of changes from another PR. At this point it may be easiest to wait on that PR to be merged rather than undo these changes.

**Comment by truongdd03** [comment]
@huss Thank you for your feedback. I have made changes to PR #668. Can you take a look and let me know if I need to fix anything. Thank you.

**Review by huss — APPROVED** [review]
With the merging of PR #668 all the concerns should be gone. I tried merging the updated development into this PR and everything looked okay and worked fine. The two failing tests are the same issue as in PR #668 and discussed there. Thanks to @truongdd03 for fixing another bar export issue.

**Pull request merged**
